### PR TITLE
[MARCHETYPES-84] Refresh download page

### PR DESCRIPTION
**Pull request by slawekjaranowski**
https://issues.apache.org/jira/browse/MARCHETYPES-84

### CODE REVIEW

File: src/site/xdoc/download.xml.vm [hunk 1]
```diff
@@ -23,102 +23,51 @@ under the License.
...
-      <p>You will be prompted for a mirror - if the file is not found on yours, please be patient, as it may take 24
-      hours to reach all mirrors.<p/>
-
-      <p>In order to guard against corrupted downloads/installations, it is highly recommended to
-      <a href="http://www.apache.org/dev/release-signing#verifying-signature">verify the signature</a>
-      of the release bundles against the public <a href="https://www.apache.org/dist/maven/KEYS">KEYS</a> used by the Apache Maven
-      developers.</p>
+      <p><strong>${project.name} ${project.version}</strong> is distributed in source format.</p>
 
-      <p>${project.name} is distributed under the <a href="http://www.apache.org/licenses/">Apache License, version 2.0</a>.</p>
```
[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply src/site/xdoc/download.xml.vm:40]: Such name is used in others ASF Maven projects, please look ag: https://maven.apache.org/download.cgi
there are:
```
Distributed under the Apache License, version 2.0
```

[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply src/site/xdoc/download.xml.vm:40]: This line is not removed, it is in other line - 36 in target

**Review by Bukama — APPROVED** [review]
Thanks for pointing me to the licences line. I totally oversaw the move in the diff :(

**Comment by jira-importer** [comment]
Resolve #119

**Pull request merged**